### PR TITLE
fixed error on attr. replace with className.

### PR DESCRIPTION
**Pull request by jcstang**
Jean, i was able to fix the dropdown. just missing the bootstrap js scripts on index.html

### CODE REVIEW

**Comment by JeanPeel** [comment]
Thanks for solving my className issue too.  It was bootstrap was not working unless I had them just set to class.  Glad you helped get that set up right!

**Comment by JeanPeel** [comment]
OMGoodness!! Code is soooo strange....!

When I paste this code into the other pages the layout fixes.

<div className='image_div2'>Candid Photo</div>

now to figure out why....

**Comment by JeanPeel** [comment]
code did not show up... let me try again...

"<div className='image_div2'>Candid Photo</div>"

**Comment by JeanPeel** [comment]
Well I have tried just using the classNames of that pic... and I have tried turning off the media calls on that pic.... and I found that in my css the media call misspelled blog as plog..... But.... Still, it has to be that image in the page or the page does not render right, it has nothing to do with class names, it has nothing to do with media calls.

It is strange... But at least it is working for now... with some extra images scattered around.

correction.... as long as I have an image on the other pages the formatting looks correct.

**Comment by jcstang** [comment]
Hey @JeanPeel did you ever get this figured out?

**Comment by JeanPeel** [comment]
Thanks for checking! I did figure out what the problem is.   It is how I put together the media calls and positioning.  Now I know better how not to do it next time.
